config.go

```diff
@@ -346,27 +346,27 @@ func loadConfig() (*config, error) {
 
 	// Ensure that the user didn't attempt to specify negative values for
 	// any of the autopilot params.
-	if cfg.Autopilot.MaxChannelSize < 0 {
-		str := "%s: autopilot.maxchansize must be greater than zero"
-		err := fmt.Errorf(str)
+	if cfg.Autopilot.MaxChannels < 0 {
+		str := "%s: autopilot.maxchannels must be greater than zero"
```

Comment:
This should read `autopilot.maxchannels must be non-negative`. While we're at it, can you fix this for the 3 other checks below as well?